helm/metrics-server-app/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 appVersion: 0.3.3
 description: Metrics Server is a cluster-wide aggregator of resource usage data.
-name: kubernetes-metrics-server-chart
+name: metrics-server-app
```

Comment:
Let's bump the version below to 0.4.0 and add a changelog entry. See https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md.